code/modules/food_and_drinks/food/snacks_egg.dm

```diff
@@ -124,7 +124,6 @@
 	bonus_reagents = list(/datum/reagent/consumable/nutriment = 1, /datum/reagent/consumable/nutriment/vitamin = 2)
 	list_reagents = list(/datum/reagent/consumable/nutriment = 8, /datum/reagent/consumable/nutriment/vitamin = 1)
 	bitesize = 1
-	w_class = WEIGHT_CLASS_NORMAL
```

Comment:
Don't remove this.

Reply:
Forgot to restore that with the whitelisting.